testdata/project-v2/api/v1/admiral_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023 The Kubernetes authors.
+Copyright 2024 The Kubernetes authors.
```

Comment:
@lukas016 could you pleas rebase with master ?
It is merged already

Reply:
hi, sorry i did rebase with my current master. It should be fixed now